### PR TITLE
Fixed an invalid entry and added citation.

### PR DESCRIPTION
**Pull request by m1cm1c**

### CODE REVIEW

File: doc/Requirements Specification/Render.sh [hunk 1]
```diff
@@ -5,6 +5,9 @@ file="Requirements Specification"
 # fail the script if any command fails
 set -e 
 
+# Delete old rendered file
+rm "Requirements Specification.pdf"
```
Comment (jGleitz) [inline doc/Requirements Specification/Render.sh]: Why should we need this line?

File: doc/Requirements Specification/Bibliography.bib.bak [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+% Encoding: Cp1252
+
+
+@Electronic{tuChemPcm,
+  Title                    = {Palladio Component Model},
+  Month                    = {09},
+  Url                      = {https://www.tu-chemnitz.de/informatik/ST/research/projects/palladio-component-model},
+  Year                     = {2015}
+}
+
```
Comment (jGleitz) [inline doc/Requirements Specification/Bibliography.bib.bak]: You included the `.bak` file. Please exclude it.

Consider excluding `**/*.bak` in the global `.gitignore`

File: doc/Requirements Specification/Render.sh (hunk 1 above)
Comment (jGleitz) [inline doc/Requirements Specification/Render.sh]: Plus, it fails, if `Requirement Specifications.pdf` doesn’t exist, as you can see in your failing Travis build.

**Comment by m1cm1c** [comment]
> Why should we need this line?

Rendering the new file can fail with the error message that a file with the same name already exists on the ext4 file system.

**Comment by jGleitz** [comment]
> Rendering the new file can fail with the error message that a file with the same name already exists on the ext4 file system.

Are you sure that the existence of the file in the project folder is the cause here? I’d rather expect some error in the temporary folder.

The rendered file is copied to the project folder at the very end of the render process, using `cp -rf`.

I don’t like the `rm -f`. If the rendering fails, you’ll have no PDF anymore.

**Comment by jGleitz** [comment]
The `.bak` is still part of your PR!

**Comment by jGleitz** [comment]
lgtm

**Comment by m1cm1c** [comment]
Neither of these flags exist (and `-r` doesn't do anything when copying a single file anyway). But I added the `-f` flag. The backup file is removed.

**Comment by jGleitz** [comment]
> Neither of these flags exist

I just realised that. Thanks for adding.

> and -r doesn't do anything when copying a single file anyway

so true.

**Comment by acBerger** [comment]
> This branch is out-of-date with the baster branch

@m1cm1c: please merge latest changes into this branch

**Comment by m1cm1c** [comment]
done

**Comment by acBerger** [comment]
lgtm

**Comment by jGleitz** [comment]
@m1cm1c, what did you do to duplicate all commits on this PR?

**Comment by jGleitz** [comment]
Forget about it, they didn’t make it into master, so I just assume GitHub was acting weird.

**Pull request merged**